### PR TITLE
`state import` uses a private namespace if added packages do not have one.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2941" title="DX-2941" target="_blank">StoryDX-2941</a>  Users can import and create projects from CycloneDX SBOMs using `state import`
  </td></table>
  <br />
 

This can happen with language-agnostic formats like CycloneDX and SPDX.

### CODE REVIEW

File: internal/runbits/org/org.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org
```
Comment (mitchell-as) [inline internal/runbits/org/org.go:1]: This comes from the init runner, which selects a default org if one wasn't specified. This code is not new.

**Comment by mitchell-as** [comment]
Test failures are known issues and not due to this PR.

File: internal/runbits/org/org.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	if desiredOrg != "" {
+		// Match the case of the organization.
+		// Otherwise the incorrect case will be written to the project file.
+		for _, org := range orgs {
+			if strings.EqualFold(org.URLname, desiredOrg) {
+				return org.URLname, nil
+			}
+		}
+		// Return desiredOrg for error reporting
+		return desiredOrg, ErrNoOwner
```
Comment (MDrakos) [inline internal/runbits/org/org.go:39]: I believe that in go if an error is returned the other return value is not to be trusted. I think a better approach here may be to use an error type and have the desired org be a member of it.

**Review by MDrakos — CHANGES_REQUESTED** [review]

File: internal/runbits/org/org.go (hunk 2 above)
Comment (mitchell-as) [reply internal/runbits/org/org.go]: Technically, fixing this is out of scope for the ticket (this is not new code), but I've gone ahead and done so.

Comment (MDrakos) [reply internal/runbits/org/org.go]: Thank you! 🙏

**Review by MDrakos — APPROVED** [review]

**Pull request merged**